src/cli/repo/repo.go

```diff
@@ -17,6 +17,7 @@ import (
 const (
 	initCommand        = "init"
 	connectCommand     = "connect"
+	updateCommand      = "update-passphrase"
```

Comment:
nit: since the command is now specific to updating the passphrase could you do a quick search and replace on this and other similarly named variables to start with `updatePassphrase` please?